CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Routing
 
 * Added `Router.finishRouting()` method to finish routing session without dismissing related UI and logic components. ([#3880](https://github.com/mapbox/mapbox-navigation-ios/pull/3880))
+* Added reporting `AlternativeRoutes` during navigation sessions. Subscribe to `RouterDelegate.router(_:didUpdateAlternatives:removedAlternatives:)` or `routeControllerDidUpdateAlternatives` notification to receive updates on actual alternative routes list. Switching to the alternative route could be monitored by observing `RouterDelegate.router(_:willTakeAlternativeRoute:at:)` and `RouterDelegate.router(_:didTakeAlternativeRouteAt:)` methods or `routeControllerWillTakeAlternativeRoute` and `routeControllerDidTakeAlternativeRoute` notifications. To configure alternative routes detection, setup `NavigationSettings.alternativeRouteDetectionOptions` before starting navigation. ([#3833](https://github.com/mapbox/mapbox-navigation-ios/pull/3833))
```

Comment:
`AlternativeRoutes` isn’t the name of a type. Either `AlternativeRoute`s or “alternative routes”.

jazzy needs the fully qualified name `Notification.Name.routeControllerDidUpdateAlternatives`.